keyboards/donutcables/scrabblepad/config.h

```diff
@@ -19,167 +19,21 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #include "config_common.h"
 
-/* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x0000
-#define DEVICE_VER      0x0001
-#define MANUFACTURER    Donut Cables
+#define VENDOR_ID       0x4443
+#define PRODUCT_ID      0xSCPD
```

Comment:
This isn't a valid entry, it HAS to be a hex value. 

```
❯ echo -n "ScrabblePad" | shasum | cut -c1-4 | tr '[:lower:]' '[:upper:]'
21D7
```

```suggestion
#define PRODUCT_ID      0x21D7
```